pal_event_server/test/dbus_notifications_test.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// @dart=2.9
+
+import 'package:test/test.dart';
+import 'package:pal_event_server/src/daemon/linux/dbus_notifications.dart';
+
+void main() {
+  group('All', ()
+  {
+    test('notificationWithEmptyEventAndNoNotificationsOutstanding', () {
+      notifications = <int, int>{};
+      listen("");
```

Comment:
Could we add some comments here about expected behavior, such as expected to do nothing and generate no errors?

Reply:
Added descriptions. Please take a look. Let me know if this is what you had in mind?